isyfact-standards-bom/pom.xml

```
@@ -15,8 +15,8 @@
     <description>Definiert die Versionen aller Bibliotheken der IsyFact-Standards.</description>
 
     <properties>
-        <isy-exception-sst.version>1.11.1</isy-exception-sst.version>
-        <isy-serviceapi-sst.version>1.11.1</isy-serviceapi-sst.version>
+        <isy-exception-sst.version>1.10.0</isy-exception-sst.version>
+        <isy-serviceapi-sst.version>1.9.0</isy-serviceapi-sst.version>
```

Warum wird hier nicht auch die 1.10.0 genutzt?
```suggestion
        <isy-serviceapi-sst.version>1.10.0</isy-serviceapi-sst.version>
```

---

Die habe ich wohl übersehen. Version 1.10.0 wird jetzt verwendet.